packages/serverless/ci/pipeline.yaml

```diff
@@ -294,7 +294,7 @@ jobs:
         file: instana-aws-lambda-npm-package/version
         reveal: true
 
-      - task: build-and-publish-aws-lambda-image
+      - task: build-and-publish-aws-lambda-image-x86_64
```

Comment:
_nitpick:_ I think we previously mentioned that the current naming is slightly misleading (my bad). How about we  rename the job (line 283) from `aws-lambda-layer-as-container-image` to something like `aws-lambda-layer-and-container-image` to be more clear that it does both, layer and container image? And maybe rename the two tasks to `build-and-publish-aws-lambda-layer-and-image-$architecture`?

Reply:
Oh yeah good idea. 

> build-and-publish-aws-lambda-layer-and-image-$architecture?

👍